scripts/isolinux.cfg

```diff
@@ -1,5 +1,9 @@
-default rancheros
-label rancheros
-    kernel /boot/vmlinuz
-    initrd /boot/initrd
-    append quiet rancher.autologin=tty1 rancher.autologin=ttyS0
+TIMEOUT 20  #2s
+#PROMPT 1
+
+# doesn't appear to work here?
+INCLUDE ../global.cfg
+
+# each INCLUDEd file has a `DEFAULT mylabel` in it, and the last one wins
+INCLUDE ../linux-previous.cfg
```

Comment:
We set the `autologin` parameter on the ISO though, so I'm not sure how we'd share the config. I don't think I understand the recovery part either - why would an ISO have a recovery option?